src/lib/y2storage/partitionable.rb

```diff
@@ -43,6 +43,8 @@ class Partitionable < BlkDevice
     #   @return [Array<PartitionTables::Type>]
     storage_forward :possible_partition_table_types, as: "PartitionTables::Type"
 
+    storage_forward :default_partition_table_type, as: "PartitionTables::Type"
+
```

Comment:
What I would do is:
1) Not exposing `default_partition_table_type` publicly, but define it as private
2) Moving `preferred_ptable_type` to `Partitionable`, implementing it as a direct forward to the private method (maybe just an alias).
3) Re-implementing it in `Disk` to always offer GPT.